src/openrct2/ride/RideData.cpp

```diff
@@ -322,7 +322,7 @@ bool RideTypeDescriptor::HasFlag(uint64_t flag) const
 
 void RideTypeDescriptor::GetAvailableTrackPieces(RideTrackGroup& res) const
```

Comment:
Why the fuck does it even pass in a RideTrackGroup instead of just returning something? Never seen this code.